Assets/_Project/Scripts/SpectrobeCard.cs

```diff
@@ -10,16 +11,27 @@ namespace SpectrobesTCG
 
     public class SpectrobeCard : Card
     {
-        public CardManager.SpectrobeForm form = CardManager.SpectrobeForm.adult;
-        public CardManager.SpectrobeProperty property = CardManager.SpectrobeProperty.aurora;
-        public int attack;
-        public int hp;
-
-
+        public SpectrobeData spectrobeData;
```

Comment:
This card in particular will probably need to reference both the Adult and Evolved spectrobe's Data. We don't need to assign it just yet because we don't have the prefab setup to show those values but just to prepare in the future.

Reply:
I'm not sure I did what you had in mind but now adult cards can store the info from 2 different SOs, one for the adult form and one for the evolved, therefore we'll have access to the stats of both cards at all times